api/src/pcapi/routes/pro/bookings.py

```diff
@@ -114,6 +115,13 @@ def get_bookings_pro(query: ListBookingsQueryModel) -> ListBookingsResponseModel
     )
 
 
+@blueprint.pro_private_api.route("/bookings/pro/userHasBookings", methods=["GET"])
+@login_required
+def get_offerer_has_bookings() -> bool:
```

Comment:
Tu regarde si un utilisateur à des reservations pour au moins un de ces "user offerer".
plutot 
```suggestion
def get_user_has_bookings() -> bool:
```